packages/mdc-animation/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.js
+!*.ts
+# must come after !*.ts
+*.d.ts
+*.js.map
```

Comment:
Can any of these rules go in the root `.gitignore` / `.npmignore` file? (Or `packages/.gitignore`)

We can always override them for specific packages if necessary (e.g., `mdc-base` or `mdc-dom` have a `.d.ts` file, or packages that haven't been converted to TypeScript yet).

Reply:
For `.gitignore` we should be able to make it common, e.g. `packages/**/*.js`. I am inclined to think this will _not_ work for `.npmignore` though.

Reply:
moved to global gitignore